doc/admin-guide/plugins/maxmind_acl.en.rst

```diff
@@ -78,3 +78,32 @@ Optional
 ========
 
 There is an optional ``html`` field which takes a html file that will be used as the body of the response for any denied requests if you wish to use a custom one.
+
+Anonymous
+=========
+
+There is also an optional ``anonymous`` field. This allows you to use the maxmind anonymous IP database and reference it's optional fields. These are ``ip``, ``vpn``, 
```

Comment:
MaxMind GeoIP2 Anonymous IP database?

Reply:
yes, meant to be used with, https://dev.maxmind.com/geoip/docs/databases/anonymous-ip?lang=en#binary-database, the maxmind anonymous IP database. It's the same format but more barebones than the normal geo since its just a handful of fields returned with no nested structure to it

Reply:
Sorry, I meant to say replace "maxmind anonymous IP database" with "MaxMind GeoIP2 Anonymous IP database". (adding capitalization to the company and product names).